elasticsearch/elasticsearch.yml

```diff
@@ -29,16 +36,14 @@ searchguard:
   allow_all_from_loopback: false
```

Comment:
the setting for index that needs to be configured per instance: searchguard.config_index_name: searchguard